source/faq.txt

```diff
@@ -1,3 +1,5 @@
+.. _kotlin-faq:
+
 ===
 FAQ
 ===
```

Comment:
S: add a note to this page linking to the connection troubleshooting guide like the java one has: https://github.com/mongodb/docs-java/blob/6b87e5d49f837d6f1d9a16909d09df58828278a5/source/faq.txt#L18-L23